content/en/docs/instrumentation/ruby/manual.md

```diff
@@ -29,70 +81,145 @@ end
 
 ### Creating New Spans
 
-Auto-instrumentation can show the shape of requests to your system, but only you know the really important parts.
-In order to get the full picture of what's happening, you will have to add manual instrumentation and create some custom spans.
-To do this, grab the tracer from the OpenTelemetry API and generate a span:
+To create a [span](/docs/concepts/signals/traces#spans-in-opentelemetry), you’ll
+need a [configured `Tracer`](#acquiring-a-tracer).
+
+Typically when you create a new span, you'll want it to be the active/current
+span. To do that, use `in_span`:
 
 ```ruby
-# ...
+require "opentelemetry/sdk"
 
-def search_by(query)
-  tracer = OpenTelemetry.tracer_provider.tracer('my-tracer')
-  tracer.in_span("search_by") do |span|
-    # ... expensive query
+def do_work()
+  MyAppTracer.in_span("do_work") do |span|
+    # do some work that the 'do_work' span tracks!
   end
 end
 ```
 
-The `in_span` convenience method is unique to Ruby implementation, which reduces some of the boilerplate code that you would have to otherwise write yourself:
+### Creating nested spans
+
+If you have a distinct sub-operation you’d like to track as a part of another
+one, you can create nested
+[spans](/docs/concepts/signals/traces#spans-in-opentelemetry) to represent the
+relationship:
 
 ```ruby
-def search_by(query)
-  span = tracer.start_span("search_by", kind: :internal)
-  OpenTelemetry::Trace.with_span(span) do |span, context|
-    # ... expensive query
+require "opentelemetry/sdk"
+
+def parent_work()
+  MyAppTracer.in_span("parent") do |span|
+    # do some work that the 'parent' span tracks!
+
+    child_work()
+
+    # do some more work afterwards
   end
-rescue Exception => e
-  span&.record_exception(e)
-  span&.status = OpenTelemetry::Trace::Status.error("Unhandled exception of type: #{e.class}")
-  raise e
-ensure
-  span&.finish
+end
+
+def child_work()
+  MyAppTracer.in_span("child") do |span|
+    # do some work that the 'child' span tracks!
```

Comment:
```suggestion
def child_work
  MyAppTracer.in_span("child") do |span|
    # do some work that the 'child' span tracks!
  end
```

Style nit, but also we missed an `end` keyword (and so it was technically syntactically invalid)